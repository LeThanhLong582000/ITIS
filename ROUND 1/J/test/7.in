47
789320303 475013305
352354112 564281719
558965524 543807371
147339685 444810328
589457424 495673017
454792508 978353501
6078083 977984046
348233460 863479291
631445597 5716808
107429982 847798127
236403633 55492331
526039528 197740457
451745351 897418617
425307301 816011542
714193381 498662279
275878038 643839597
1534923 279694189
40007901 353562778
238570185 706950200
689225325 509568456
659713216 126378547
615714172 721549308
340467927 553731673
608952622 236043865
432302775 343959094
578223395 51887546
477915776 380109825
294411943 278429325
180641215 755707576
804832388 199165612
542508742 729160298
547072659 920914494
597109362 695224402
782621884 55390872
653789811 987979479
258726970 626349803
4074127 450408068
955641340 443372002
622096318 106190624
38829850 514480091
834082889 562209693
844056195 98432346
971572852 505639302
407923463 645696305
533205404 983468743
552402622 340379365
360845514 697770114
