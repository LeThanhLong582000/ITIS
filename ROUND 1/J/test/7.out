6314562424
2818832896
4471724192
1178717480
4715659392
3638340064
48624664
2785867680
3788673582
859439856
1654825431
4208316224
3613962808
3402458408
5713547048
2207024304
12279384
320063208
1908561480
5513802600
5277705728
4925713376
2723743416
4871620976
3458422200
4047563765
3823326208
2355295544
1445129720
6438659104
4340069936
4376581272
4776874896
5478353188
5230318488
2069815760
32593016
7645130720
4976770544
310638800
6672663112
5908393365
7772582816
3263387704
4265643232
4419220976
2886764112
